src/lang/modifyAst/sweeps.ts

```diff
@@ -520,3 +619,14 @@ export function retrieveAxisOrEdgeSelectionsFromOpArg(
   }
   return { axisOrEdge, axis, edge }
 }
+
+export function retrieveTagDeclaratorFromOpArg(
+  opArg: OpArg,
+  code: string
+): string {
+  const dollarSignOffset = 1
+  return code.slice(
+    opArg.sourceRange[0] + dollarSignOffset,
+    opArg.sourceRange[1]
+  )
+}
```

Comment:
This is exactly the kind of stuff I can't wait to not need once we have a code mod API